Mathlib/Analysis/Normed/Field/Lemmas.lean

```diff
@@ -176,6 +176,9 @@ instance (priority := 100) NonUnitalSeminormedRing.toContinuousMul [NonUnitalSem
 instance (priority := 100) NonUnitalSeminormedRing.toIsTopologicalRing [NonUnitalSeminormedRing α] :
     IsTopologicalRing α where
 
+@[deprecated (since := "2025-14-02")] alias NonUnitalSeminormedRing.toTopologicalRing :=
```

Comment:
Please use the YYYY-MM-DD format, here and elsewhere. (This is not strictly enforced yet... but makes it much easier to see which deprecations are half a year old, hence can be removed.)

Reply:
Oh good catch!